docs/azure/sdk/authentication/azure-hosted-apps.md

```diff
@@ -3,14 +3,14 @@ title: Authenticating Azure-hosted apps to Azure resources with the Azure SDK fo
 description: This article covers how to configure authentication for apps to Azure services when the app is hosted in an Azure service like Azure App Service, Azure Functions, or Azure Virtual Machines.
 ms.topic: how-to
 ms.custom: devx-track-dotnet, engagement-fy23, devx-track-azurecli
-ms.date: 2/28/2023
+ms.date: 02/28/2023
 ---
 
 # Authenticating Azure-hosted apps to Azure resources with the Azure SDK for .NET
 
 When an app is hosted in Azure using a service like Azure App Service, Azure Virtual Machines, or Azure Container Instances, the recommended approach to authenticating an app to Azure resources is to use a [managed identity](/azure/active-directory/managed-identities-azure-resources/overview).
 
-A managed identity provides an identity for your app such that it can connect to other Azure resources without the need to use a secret key or other application secret.  Internally, Azure knows the identity of your app and what resources it's allowed to connect to.  Azure uses this information to automatically obtain Azure AD tokens for the app to allow it to connect to other Azure resources, all without you having to manage any application secrets.
+A managed identity provides an identity for your app such that it can connect to other Azure resources without the need to use a secret key or other application secret. Internally, Azure knows the identity of your app and what resources it's allowed to connect to. Azure uses this information to automatically obtain Microsoft Entra tokens for the app to allow it to connect to other Azure resources, all without you having to manage any application secrets.
```

Comment:
```suggestion
A managed identity provides an identity for your app such that it can connect to other Azure resources without the need to use a secret key or other application secret. Internally, Azure knows the identity of your app and what resources it's allowed to connect to. Azure uses this information to automatically obtain Microsoft Entra ID tokens for the app to allow it to connect to other Azure resources, all without you having to manage any application secrets.
```

We need the "Entra ID" for branding, no?

Reply:
I'm following the guidance at https://learn.microsoft.com/entra/fundamentals/how-to-rename-azure-ad#how-to-use-find-and-replace-for-text-based-content. See this blurb:

> Review whether each instance refers to the product or a feature of the product.
>
> - Azure AD as the product name alone should be replaced with Microsoft Entra ID.
> - Azure AD features or functionality become Microsoft Entra features or functionality. For example, "Azure AD Conditional Access" becomes "Microsoft Entra Conditional Access."